aconfigure.ac

```diff
@@ -164,6 +164,11 @@ case $target in
 	AC_DEFINE(PJ_WIN32,1)
 	AC_DEFINE(PJ_WIN32_WINNT,0x0400)
 	AC_DEFINE(WIN32_LEAN_AND_MEAN)
+	case $target in
+	    *w64-mingw* )
```

Comment:
I think this should be something like *_64-w64-mingw* instead. Because otherwise "i686-w64-mingw32" here also sets PJ_WIN64 to 1 while being 32-bit windows. And while apparently this does not break the build, I suppose the intent was to only set PJ_WIN64 for a 64-bit system.